src/Controls/src/Core/RadioButton/RadioButton.iOS.cs

```diff
@@ -8,10 +8,44 @@ public static void MapContent(RadioButtonHandler handler, RadioButton radioButto
 
 		public static void MapContent(IRadioButtonHandler handler, RadioButton radioButton)
 		{
+			// We want to modify the text in the content property only if
+			// the content is a string. In that case we want to apply
+			// the text values from the radio button
+			if (radioButton.Content is string
+				&& handler.VirtualView.PresentedContent is Element element
+				&& FindLabel(element) is Label label)
+			{
```

Comment:
With the changes in the Mapper, are not similar changes required on Android as well?

Reply:
Android radio buttons work without any changes needed. Therefore, I used the preprocessor directives `#if IOS` to modify only the iOS's mapper